### ISSUE TITLE
CF variable added as NONE thesaurus cannot be removed

### ISSUE THREAD

**Issue by heshamkelbashandy** [heshamkelbashandy/ess-dive-catalog#220]
In GitLab by @vchendrix on May 7, 2018, 11:23

Jira EDR-52

If a data variable is in the thesaurus and it is added as a variable that is not in a thesaurus it cannot be removed from the list. This happens when you don't select the variable from the suggested values in the drop down

### Variable keywords are entered as
```xml
<keywordSet>
   <keyword>latitude</keyword>
   <keyword>longitude</keyword>
   <keyword>
   Order	Transect	Plot	Lat	Long	Photos	Tree_Species	 
Tree_Diameter	Tree_Height	Seed_Sap_Species	Seed_Sap_Number	Eric_Shrub_Species	Eric_Shrub_Cover Herb_Grass_Forbes_Species	Herb_Grass_Forbes_Status	Peat_Depth	Notes	Tree_BA
   </keyword>
   <keywordThesaurus>VARIABLE:NONE</keywordThesaurus>
</keywordSet>
```

### Cannot remove latitude and longitude below
![Screen_Shot_2018-05-07_at_11.24.06_AM](/uploads/82f9a590842daf73a1eb8e8f662bdfc6/Screen_Shot_2018-05-07_at_11.24.06_AM.png)

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 7, 2018, 11:27

changed the description

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 24, 2018, 14:49

Duplicate of #103

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 24, 2018, 14:49

closed

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 24, 2018, 14:58

changed the description

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 24, 2018, 14:59

Closed via ff87b47663c9355c528f441ec05bbb2860bc49f4

**Comment by heshamkelbashandy** [comment]
In GitLab by @vchendrix on May 25, 2018, 09:26

added ~4134545 label
